docs/reference/cli-reference/cli-ref-pack.md

```diff
@@ -71,6 +71,14 @@ For example, consider the following `packages.config` file in the source project
 
 For this project, the package created by `nuget pack` will have a dependency on `jQuery` and `microsoft-web-helpers` but not `netfx-Guard`.
 
+## Suppressing pack warnings
+
+While it is recommended that you resolve all NuGet warnings during your pack operations, in certain situations suppressing them is warranted.
+
+You can achieve that in the following way: 
+
+> nuget.exe pack package.nuspec -Properties NoWarn=NU5104
+
```

Comment:
Should this also refer to the above documentation if people want to suppress them using configs instead of CLI flags? And vice-versa -- should we have a section that points from there to here?

Reply:
Can you clarify what you mean by configs?

Reply:
sorry, I mean using properties*

Reply:
Ahh...nuget.exe pack doesn't require a csproj, so you can't always specify it there. 

Technically in some situations nuget.exe pack will read the csproj and might work, but honestly, that might be too messy to explain and ideally people use dotnet.exe pack/msbuild /t:pack for csproj packing scenarios.

Reply:
Thoughts?
Please resolve the thread if you agree, or if you have a specific recommendation.